frontend/src/components/ui/question/QuestionBox.js

```diff
@@ -11,33 +11,35 @@ import QuestionCategories from "./QuestionCategories";
 import QuestionTitle from "./QuestionTitle";
 import Question from "./Question";
 
-import styles from './QuestionBox.module.css'
+import styles from "./QuestionBox.module.css";
 
-/* 
-  Find out which difficulty user chose,
-  obtain a Question of that difficulty from
-  Question service and display it
-*/
-
-function QuestionBox() {
+// Find out which difficulty user chose,
+// obtain a Question of that difficulty from
+// Question service and display it
+function QuestionBox({ difficulty, roomId }) {
   const [title, setTitle] = useState("");
   const [categories, setCategories] = useState([]);
   const [content, setContent] = useState("");
-  const [difficulty, setDifficulty] = useState("");
+  const [questionNumber, setQuestionNumber] = useState();
 
   useEffect(() => {
     async function fetchData() {
+      // Encoded roomId
+      var questionEncoding = encode(roomId);
+      setQuestionNumber(questionEncoding);
+
       await axios
-        .get("http://localhost:8004/api/question/level/Hard") // 'Hard' is hardcoded for now
+        .get(
+          "http://localhost:8004/api/question/level/" +
```

Comment:
Sorry just realized but do u wanna import the endpoint from config.js instead?

Reply:
Thanks for checking this and providing this suggestion! Much more simpler now to read!